src/linter/VerilatorLinter.ts

```diff
@@ -77,10 +83,10 @@ export default class VerilatorLinter extends BaseLinter {
                     line = line.substr(1)
 
                     // was it for a submodule
-                    if (line.search(doc.fileName) > 0)
+                    if (line.search(docUri) > 0)
```

Comment:
Why did you changed from `doc.fileName` to `docUri`?
Is it Windows specific reason?
I haven't tested on Linux nor macOS. But I'm curious that whether this change break feature on non-Windows environment or not?

Reply:
In Windows, verilator seems to recognize "/" as a file path separator. Actually, the file path in the verilator output (line variable) is also separated by "/". Therefore, in order to search and replace the file path, I used docUri variable (separated by "/") instead of doc.filename (separated by "\\").

In addition, "docUri (doc.uri.fsPath)" is equivalent to "doc.filename".
https://code.visualstudio.com/api/references/vscode-api#TextDocument

> fileName: string
> The file system path of the associated resource. Shorthand notation for TextDocument.uri.fsPath. Independent of the uri scheme.

Therefore, on Linux and MacOS, I think it's equivalent to the latest revision. However, I have not tested.

Reply:
Ok. I’ll merge it ASAP.